dask/dataframe/core.py

```diff
@@ -3137,16 +3148,21 @@ def value_counts(
                 )
             kwargs["dropna"] = dropna
 
+        meta = self._meta.value_counts()
+        if normalize:
+            meta /= self._meta.sum()
```

Comment:
Doesn't pandas already support the `normalize` argument?

Reply:
yeah I guess the meta could just use normalize.